lib/matplotlib/colorbar.py

```diff
@@ -285,6 +284,32 @@ def _set_inner_bounds(self, bounds):
         self.inner_ax._axes_locator = _TransformedBoundsLocator(
             bounds, self.outer_ax.transAxes)
 
+    def cla(self):
+        """
+        Reset the colorbar axes to be empty.
+        """
+        # need to low-level manipulate the stacks because we
+        # are just swapping places here.  We don't need to
+        # set transforms etc...
+        self.inner_ax.cla()
+        self.outer_ax.cla()
+        self.figure._axstack.add(self)
+        self.figure._axstack.remove(self.outer_ax)
+        self.figure._localaxes.add(self)
+        self.figure._localaxes.remove(self.outer_ax)
+
+        self.__dict__.update(self.outer_ax.__dict__)
+        self.inner_ax.remove()
+        del self.inner_ax
+        del self.outer_ax
+
+        self.xaxis.set_visible(True)
```

Comment:
Here self is back to an Axes now, so can you call `super().cla()` to get the defaults for you?